docs/proof-of-stake/ticket-splitting.md

```diff
@@ -0,0 +1,112 @@
+# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> Ticket Splitting
+
+---
+
+Ticket splitting is the process of splitting the cost of a full ticket with other DCR holders. This allows you to stake and participate in Decred's governance even if you don't have enough DCR to purchase a full ticket. The minimum amount of DCR that can be put into a split ticket is 5 DCR. 
+
+This page will provide you a high-level description of how ticket splitting works, instructions on setting up the necessary Split Ticket Buyer software and splitting your first ticket. 
+
+If you prefer video, try this [tutorial](https://www.youtube.com/watch?v=9L8P7hL5v6w).
+
+!!! warning "Warning"
+    Ticket Splitting is still in Beta. The code for splitting tickets has not been extensively vetted by automated tests and more experienced community members. See [risks in joining the beta](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/beta.md#risks-in-joining-the-beta) for further explanation of the risks. 
+
+
+## Requirements
+
+- **Decrediton wallet:** You will need [Decrediton](../wallets/decrediton/decrediton-setup.md) version 1.2.1 or later to run the Split Ticket Buyer. 
+- **5 DCR:** A minimum of 5 DCR is required to participate in ticket splitting. This must be in your wallet, not on an exchange. 
```

Comment:
Slightly more than 5 actually, to account for transaction fees.

Reply:
Since the GUI can only do integers, does this mean you have to pledge 6 DCR? Or would the transaction fees be automatically added to the 5 DCR?

Reply:
They are added, so you need an amount _in excess_ of whatever you put as a pledge. For example, if you put in 5 dcr, you'll actually use some 5.003 (amount will actually vary depending on how many inputs you'll actually send once the session starts). So anything above 5.1 should be more than enough.

Reply:
Took a shot at rewording. Is this accurate?

> The minimum amount of DCR you can contribute is 5 DCR. Transaction fees will be automatically calculated by the Split Ticket Buyer and added to your pledged amount when the ticket is purchased. Transaction fees will vary based on the number of participants in the split ticket. A typical transaction fee might be 0.003 DCR. If you have more than 0.1 DCR in your wallet for fees, that should be more than enough.

Reply:
Seems fair enough.